chart/values.yaml

```diff
@@ -30,7 +30,7 @@ replicaCount: 3
 image:
   repository: longhornio/upgrade-responder
   tag: latest
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
```

Comment:
Don't do this. We will have a release once everything settled down. Otherwise, the deployment might be broken when the upstream image updated.

Reply:
I see. Removed it. Thank you